src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -283,7 +293,30 @@ public int order() {
                 if(!dlsFlsValve.invoke(request, listener, pres.getAllowedFlsFields(), pres.getMaskedFields(), pres.getQueries())) {
                     return;
                 }
-                chain.proceed(task, action, request, listener);
+                final CreateIndexRequest createIndexRequest = pres.getRequest();
```

Comment:
Will this always be CreateIndexRequest when reached here? What about bulk request?

Reply:
It will be always `CreateIndexRequest`

Reply:
Just wondering as I saw couple of bulk request reached here. What happens to those? Not sure if you noticed the bulk request

Reply:
It is not the original request.